secedgar/cik_lookup.py

```diff
@@ -184,3 +178,34 @@ def _validate_lookup(lookup):
         """
         if not (lookup and isinstance(lookup, str)):
             raise TypeError("Lookup value must be string. Given type {0}.".format(type(lookup)))
+        return lookup
+
+    def get_ciks(self):
+        """Validate lookup values and return corresponding CIKs.
+
+        Returns:
+            ciks (dict): Dictionary with lookup terms as keys and CIKs as values.
+
+        """
+        ciks = {}
+        to_lookup = set(self.lookups)
+
+        cik_map = get_cik_map()
+
+        # all keys upper case
+        ticker_map = cik_map["ticker"]
+        title_map = cik_map["title"]
+
+        for lookup in to_lookup:
+            if lookup.upper() in ticker_map:
+                ciks[lookup] = ticker_map[lookup.upper()]
+            elif lookup.upper() in title_map:
+                ciks[lookup] = title_map[lookup.upper()]
+            else:
+                try:
+                    result = self._get_cik(lookup)
```

Comment:
can we rename `_get_cik` to be clear that this is using bs4 / screen scraping.

Reply:
What do you suggest we rename to?

Reply:
get_cik_from_html